### PR TITLE
iOS 13 support

### PR DESCRIPTION
**Pull request by lisional**
Update project and sample app to use swift 5
Modify layouts to support Dark mode

item preview is broken but this is due to iOS 13 simulator

### CODE REVIEW

**Review by Idefix60 — APPROVED** [review]
Awesome :)

File: FileBrowser/FileBrowser.swift [hunk 1]
```diff
@@ -62,7 +62,11 @@ open class FileBrowser: UINavigationController {
                                                             showSize: showSize)
 
         self.init(rootViewController: fileListViewController)
-        self.view.backgroundColor = UIColor.white
+        if #available(iOS 13, *) {
+            self.view.backgroundColor = UIColor.systemBackground
+        } else {
+            self.view.backgroundColor = UIColor.white
+        }
```
Comment (Idefix60) [inline FileBrowser/FileBrowser.swift]: Add UIColor extension with the function:
func fileBrowserBackground -> UIColor {
   if #available(iOS 13, *) {
      return .systemBackground
   } else {
      return .white 
   }
}

**Review by Idefix60 — CHANGES_REQUESTED** [review]

File: FileBrowser/FileBrowser.swift (hunk 1 above)
Comment (lisional) [reply FileBrowser/FileBrowser.swift]: You're right!
Thanks for the suggestion

Comment (lisional) [reply FileBrowser/FileBrowser.swift]: Good to see you here by the way 🎉

**Review by Idefix60 — APPROVED** [review]

**Pull request merged**